pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools", "wheel", "cython"]
```

Comment:
Yeah we dont need this here. ```crick```'s ```setup.py``` imports ```cython``` and ```numpy``` rather than declaring that as a dependency. So it expects that both are installed before we install ```crick```. We need to think how to get around that.